CHANGELOG.md

```diff
@@ -1,5 +1,19 @@
 # CHANGELOG
 
+# Version 3.2.0-BETA.3
+* Log all available jmx metrics at debug level.
+* Misc logging improvements:
+    - Adds PID to startup message. This will help support customers who need to take a thread/heap dump for any reason.
+    - Clearer timezone identifier on logs (-07:00 instead of -07).
+    - add applicationinsights.testing.etw.disabled setting just for use during testing.
+* Add grizzly instrumentation as preview.
+* Use tmpdir to store temporary jfr recordings.
```

Comment:
what is jfr?

Reply:
Java flight recorder.  Might be useful for profiler service customers

Reply:
please be explicit.  or add a link for that.